app/src/main/res/values/strings.xml

```diff
@@ -714,4 +714,17 @@
     <string name="night_theme_summary">Select your favorite night theme — %s</string>
     <string name="select_night_theme_toast">You can select your favorite night theme below</string>
     <string name="download_has_started">Download has started</string>
+    <string name="metadata_category">Category</string>
+    <string name="metadata_tags">Tags</string>
+    <string name="metadata_licence">Licence</string>
+    <string name="metadata_privacy">Privacy</string>
+    <string name="metadata_age_limit">Age limit</string>
+    <string name="metadata_language">Language</string>
+    <string name="metadata_support">Support</string>
+    <string name="metadata_host">Host</string>
+    <string name="metadata_thumbnail_url">Thumbnail url</string>
```

Comment:
```suggestion
    <string name="metadata_thumbnail_url">Thumbnail URL</string>
```

Reply:
This will be made uppercase anyway, it makes no difference, but I will change it